doc/ProfilesJSONDocumentation.md

```diff
@@ -0,0 +1,57 @@
+| **profiles** | | **Array[Object]** | **Properties listed below are specific to each unique profile.** |
+| `acrylicOpacity` | _Required_ | Number | When `useAcrylic` is set to `true`, it sets the transparency of the window for the profile. Accepts values from 0-1. |
+| `background` | _Required_ | String | Sets the background color of the profile. Overrides `background` set in color scheme if `colorscheme` is set. Uses hex color format: `"#rrggbb"`. |
+| `closeOnExit` | _Required_ | Boolean | When set to `true`, the selected tab closes when `exit` is typed. When set to `false`, the tab will remain open when `exit` is typed. |
+| `colorscheme` | _Required_ | String | Name of the terminal color scheme to use. Color schemes are defined under `schemes`. |
+| `commandline` | _Required_ | String | Executable used in the profile. |
+| `cursorColor` | _Required_ | String | Sets the cursor color for the profile. Uses hex color format: `"#rrggbb"`. |
+| `cursorShape` | _Required_ | String | Sets the cursor shape for the profile. Possible values: `"vintage"`, `"bar"`, `"underscore"`, `"filledBox"`, `"emptyBox"` |
+| `fontFace` | _Required_ | String | Name of the font face used in the profile. |
+| `fontSize` | _Required_ | Integer | Sets the font size. |
+| `guid` | _Required_ | String | Unique identifier of the profile. Written in registry format: `"{00000000-0000-0000-0000-000000000000}"`. |
+| `historySize` | _Required_ | Integer | The number of lines above the ones displayed in the window you can scroll back to. |
+| `name` | _Required_ | String | Name of the profile. Displays in the dropdown menu. |
+| `padding` | _Required_ | String | Sets the padding around the text within the window. Can have three different formats: `"#"` sets the same padding for all sides, `"#, #"` sets the same padding for top-bottom and right-left, and `"#, #, #, #"` sets the padding individually for top, right, bottom, and left. |
+| `snapOnInput` | _Required_ | Boolean | When set to `true`, the window will scroll to the command input line when typing. When set to `false`, the window will not scroll when you start typing. |
+| `startingDirectory` | _Required_ | String | The directory the shell starts in when it is loaded. |
+| `useAcrylic` | _Required_ | Boolean | When set to `true`, the window will have an acrylic background. When set to `false`, the window will have a plain, untextured background. |
+| `colorTable` | Optional | Array[String] | Array of colors used in the profile if `colorscheme` is not set. Colors use hex color format: `"#rrggbb"`. Ordering is as follows: `[black, red, green, yellow, blue, magenta, cyan, white, bright black, bright red, bright green, bright yellow, bright blue, bright magenta, bright cyan, bright white]` |
+| `cursorHeight` | Optional | Integer | Sets the height of the cursor. Only works when `cursorShape` is set to `"vintage"`. Accepts values from 25-100. |

```

Comment:
What does 25-100 mean? Clarify.

(It's the percentage of the line height from the bottom. 25 is 25% up from the bottom of the line, 100% is actually pretty much the same as the "filledBox" option).